text/0000-otlp-over-messaging-systems.md

```diff
@@ -0,0 +1,110 @@
+# OTLP over messaging systems
+
+Best practices for transporting OTLP data over a message system.
```

Comment:
Do we have any performance best practices for message systems based on the existing Collector Kafka implementation? For example should you prefer to batch into many small messages or into fewer large messages?

Reply:
I would not make it part of the spec, just like it is not part of the OTLP/gRPC spec.